src/test/java/ch/liquidmind/inflection/grammar/CommentTest.java

```diff
@@ -0,0 +1,77 @@
+package ch.liquidmind.inflection.grammar;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import ch.liquidmind.inflection.compiler.util.InflectionCompilerTestUtility;
+import ch.liquidmind.inflection.test.AbstractInflectionTest;
+
+@RunWith( Parameterized.class )
+public class CommentTest extends AbstractInflectionTest
+{
+
+	@Parameters( name = "{index}: File content: {0}, expected compilation: {1}" )
+	public static Collection< Object[] > data()
+	{
+		return Arrays.asList( new Object[][] { { "// comment", true },
+
+				{ "/* comment */", true },
+
+				{ "/* comment", true }, // TODO <inflection-error/> missing comment end, should not compile: true -> false
+
+				{ "/* comment begin" + System.lineSeparator() + " comment end */", true },
+
+				{ "/* comment begin" + System.lineSeparator() + " comment end", true }, // TODO <inflection-error/> missing comment end, should not compile: true -> false
+
+				{ "/* // */", true },
+
+				{ "/* //", true }, // TODO <inflection-error/> missing comment end, should not compile: true -> false
+
+				{ "// /*", true },
+
+				{ "// */", true },
+
+				{ "taxonomy A { /* }", false },
```

Comment:
Please comment on the justification for repeating the comment tests in this context.